stock_move_backdating/stock.py

```diff
@@ -81,10 +81,10 @@ def on_change_date_backdating(self, cr, uid, ids, date_backdating,
         if (dt > NOW):
             warning = {'title': _('Error!'), 'message': _(
                 'You can not process an actual movement date in the future.')}
-            values = {
-                'date_backdating': NOW.strftime(DEFAULT_SERVER_DATETIME_FORMAT)
-            }

```

Comment:
Doesn't it actually break the functionality of the module? As the values aren't returned the `date_backdating` will remain unchanged and it will allow the user to set a date in the future. Yet the module tells us that we cannot do that... that seems strange.